subsys/bluetooth/mesh/Kconfig

```diff
@@ -8,6 +8,26 @@ if BT_MESH
 
 menu "Bluetooth mesh"
 
+if BT_MESH_ADV_EXT
```

Comment:
Should we add that SDC need to be used for these values to change default.

Reply:
Adding `if BT_LL_SOFTDEVICE` won't work for nRF53, because on nRF53 there is no SDC (BT_CTLR) on the app core.

Reply:
Well, this is implicit downstream. The only controller qualified downstream is SDC.